apigee_edge.module

```diff
@@ -108,6 +114,41 @@ function apigee_edge_user_format_name_alter(&$name, AccountInterface $account) {
   }
 }
 
+/**
+ * Prepares variables for Apigee entity templates.
+ *
+ * Default template: apigee-entity.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - elements: An array of elements to display in view mode.
+ */
+function template_preprocess_apigee_entity(array &$variables) {
+  $variables['view_mode'] = $variables['elements']['#view_mode'];
+  /** @var \Drupal\apigee_edge\Entity\EdgeEntityInterface $entity */
+  $entity = $variables['elements']['#entity'];
```

Comment:
This should be exposed to the template at top level, as is done by core for node, and other contrib modules, for theming convenience.

```php
/** @var \Drupal\apigee_edge\Entity\EdgeEntityInterface $entity */
$variables['entity'] = $variables['elements']['#entity'];
```